x/ibc/24-host/validate.go

```diff
@@ -66,11 +80,21 @@ func PortIdentifierValidator(id string) error {
 func NewPathValidator(idValidator ValidateFn) ValidateFn {
 	return func(path string) error {
 		pathArr := strings.Split(path, "/")
+		if len(pathArr) > 0 && pathArr[0] == path {
+			return sdkerrors.Wrapf(ErrInvalidPath, "path %s doesn't contain any separator '/'", path)
+		}
+
 		for _, p := range pathArr {
-			// Each path element must either be valid identifier or alphanumeric
-			err := idValidator(p)
-			if err != nil && !sdk.IsAlphaNumeric(p) {
-				return sdkerrors.Wrapf(ErrInvalidPath, "path %s contains invalid identifier or non-alphanumeric path element: %s", path, p)
+			// NOTE: for some reason an empty string is added after Split
```

Comment:
Split does not add anything extra.

Reply:
This is because of paths that end with "/" the sep:
https://play.golang.org/p/fEOMCd71-L9

Reply:
It'll happen if the first or last token is `/`. We can either strip trailing/leading `/` or simply keep the code as-is.